src/components/molecules/banner-card/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import Actionables, { ActionType } from "../../molecules/actionables"
+
+type BannerCardProps = {
+  actions?: ActionType[]
+  title: string
+  thumbnail?: string
+} & React.RefAttributes<HTMLDivElement>
+
+type BannerCardDescriptionProps = {
+  cta?: {
+    label: string
+    onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+  }
+}
+
+const BannerCard: React.FC<BannerCardProps> & {
+  Description: React.FC<BannerCardDescriptionProps>
+  Footer: React.FC
+} = ({ title, thumbnail, actions, children }) => {
+  return (
+    <div className="rounded-rounded border bg-grey-0 border-grey-20 w-full p-xlarge">
```

Comment:
I suggest we have different paddings for different screen sizes, for example: `medium:p-xlarge xsmall:p-base`. To check with Ludvig of course, but I think we can eat up some of the paddings when we're on small screens

Reply:
Agree, the component will still need some additional work once we implement a truly responsinve layout, as it is a bit cramped on mobile screens even with adjustments to padding. But I think this is a good way to make it less cramped on a smaller computer screen/browser window :smile

Reply:
Agree, the component will still need some additional work once we implement a truly responsinve layout, as it is a bit cramped on mobile screens even with adjustments to padding. But I think this is a good way to make it less cramped on a smaller computer screen/browser window 😄